src/main/java/life/qbic/openbis/openbisclient/helper/OpenBisClientHelper.java

```diff
@@ -99,4 +104,12 @@ public static ExperimentTypeFetchOptions fetchExperimentTypesCompletely() {
     return experimentTypeFetchOptions;
   }
 
+  public static List<PropertyType> getPropertiesOfEntityType(IEntityType type) {
+  List<PropertyType> res = new ArrayList<>();
+  List<PropertyAssignment> assignments = type.getPropertyAssignments();
+  for (PropertyAssignment as : assignments) {
+    res.add(as.getPropertyType());
+  }
```

Comment:
```suggestion
  for (PropertyAssignment propertyAssignment : assignments) {
    res.add(propertyAssignment.getPropertyType());
  }
```
I would avoid using an English word such as `as`